src/Apps/Authentication/Utils/helpers.ts

```diff
@@ -176,23 +177,17 @@ export const maybeUpdateRedirectTo = (
 }
 
 export const setCookies = options => {
-  const { afterSignUpAction, destination, submissionId } = options
+  const { afterSignUpAction, destination } = options
 
   if (afterSignUpAction) {
-    Cookies.set("afterSignUpAction", JSON.stringify(afterSignUpAction))
+    Cookies.set(AFTER_AUTH_ACTION_KEY, JSON.stringify(afterSignUpAction))
   }
 
   if (destination) {
     Cookies.set("destination", destination, {
       expires: 60 * 60 * 24,
     })
   }
-
-  if (submissionId) {
-    Cookies.set("submissionId", submissionId, {
-      expires: 60 * 60 * 24,
-    })
-  }
```

Comment:
the reason we have `submissionId` stuff here is to connect an anonymous user who signs up to artsy to their submission. the scenario is: 

- user makes a submission anonymously
- user gets an email encouraging to sign up to Artsy to see their submission on MyCollection (the email includes a link with `submissionId` in the params)
- we set `submissionId` cookie 
- after they sign up, we check if there's a `submissionId` cookie 
- reach out to Convection to add the user to the submission

Reply:
I'm not entirely sure if this is still the case since CX took over this flow for a while ago. maybe @MounirDhahri is aware of any changes.

Reply:
yes, that's still the case